modules/core/src/weaver/Expectations.scala

```diff
@@ -6,6 +6,24 @@ import cats.data.{ NonEmptyList, Validated, ValidatedNel }
 import cats.effect.Sync
 import cats.syntax.all._
 
+trait Expectable[F[_], A] {
```

Comment:
I'm not sure this should be in `core`, since we only use it for scalacheck. Maybe we should rename it to `Prop`

Reply:
May be we can move it into `Checkers` itself?

Reply:
in the companion object, sure